### DISCUSSION TITLE
Differences between installing with operator than with helm chart

### DISCUSSION THREAD

**Discussion by camaeel** [bank-vaults/bank-vaults#2434] [Q&A] [answered]
Hi!

I remember I saw somewhere information what are benefits of using bank-vault's vault-operator to install vault instead of using bank-vault's helm chart. But now I can't find it. 
Is it maybe option to initialize vault secrets and manage them through vault CRD or was it something else?

**Comment by akijakya** [comment, marked as answer]
Hi @camaeel, thanks for trying out Bank-Vaults! The main difference is that the operator uses a CRD with which you can continuously configure your Vault instance(s). So you can change (delete from or add to) the config later on and the operator makes sure these changes are applied to Vault.
